plugin/src/main/java/net/pl3x/map/configuration/Advanced.java

```diff
@@ -543,87 +547,88 @@ Setting a color to black (#000000) will make it invisible.""")
+        put(Blocks.KELP, 0x578B2C);
         put(Blocks.LADDER, 0x000000);
         put(Blocks.LANTERN, 0x6A5B54);
         put(Blocks.LAPIS_BLOCK, 0x1F438C);
         put(Blocks.LAPIS_ORE, 0x68758F);
         put(Blocks.LARGE_AMETHYST_BUD, 0xA582CD);
-        put(Blocks.LARGE_FERN, 0x000000);
-        put(Blocks.LAVA, 0xD45A12);
+        put(Blocks.LARGE_FERN, 0x7D4536);
         put(Blocks.LAVA_CAULDRON, 0x4A494A);
+        put(Blocks.LAVA, 0xD45A12);
         put(Blocks.LECTERN, 0xAE8953);
         put(Blocks.LEVER, 0x000000);
-        put(Blocks.LIGHT, 0x000000);
-        put(Blocks.LIGHT_BLUE_BANNER, 0x8F7748);
+        put(Blocks.LIGHT_BLUE_BANNER, 0x000000);
         put(Blocks.LIGHT_BLUE_BED, 0x3AAFD9);
+        put(Blocks.LIGHT_BLUE_CANDLE_CAKE, 0x000000);
         put(Blocks.LIGHT_BLUE_CANDLE, 0x238BC5);
-        put(Blocks.LIGHT_BLUE_CANDLE_CAKE, 0x238BC5);
         put(Blocks.LIGHT_BLUE_CARPET, 0x3AAFD9);
-        put(Blocks.LIGHT_BLUE_CONCRETE, 0x2489C7);
         put(Blocks.LIGHT_BLUE_CONCRETE_POWDER, 0x4AB5D5);
+        put(Blocks.LIGHT_BLUE_CONCRETE, 0x2489C7);
         put(Blocks.LIGHT_BLUE_GLAZED_TERRACOTTA, 0x60A6D1);
         put(Blocks.LIGHT_BLUE_SHULKER_BOX, 0x33A6D5);
+        put(Blocks.LIGHT_BLUE_STAINED_GLASS_PANE, 0x000000);
         put(Blocks.LIGHT_BLUE_STAINED_GLASS, 0x6699D8);
-        put(Blocks.LIGHT_BLUE_STAINED_GLASS_PANE, 0x6293D0);
         put(Blocks.LIGHT_BLUE_TERRACOTTA, 0x716D8A);
-        put(Blocks.LIGHT_BLUE_WALL_BANNER, 0x8F7748);
+        put(Blocks.LIGHT_BLUE_WALL_BANNER, 0x000000);
         put(Blocks.LIGHT_BLUE_WOOL, 0x3AAFD9);
-        put(Blocks.LIGHT_GRAY_BANNER, 0x8F7748);
+        put(Blocks.LIGHT_GRAY_BANNER, 0x000000);
         put(Blocks.LIGHT_GRAY_BED, 0x8E8E87);
+        put(Blocks.LIGHT_GRAY_CANDLE_CAKE, 0x000000);
         put(Blocks.LIGHT_GRAY_CANDLE, 0x787A72);
-        put(Blocks.LIGHT_GRAY_CANDLE_CAKE, 0x787A72);
         put(Blocks.LIGHT_GRAY_CARPET, 0x8E8E87);
-        put(Blocks.LIGHT_GRAY_CONCRETE, 0x7D7D73);
         put(Blocks.LIGHT_GRAY_CONCRETE_POWDER, 0x9B9B94);
+        put(Blocks.LIGHT_GRAY_CONCRETE, 0x7D7D73);
         put(Blocks.LIGHT_GRAY_GLAZED_TERRACOTTA, 0x91A7A9);
         put(Blocks.LIGHT_GRAY_SHULKER_BOX, 0x7F7F76);
+        put(Blocks.LIGHT_GRAY_STAINED_GLASS_PANE, 0x000000);
         put(Blocks.LIGHT_GRAY_STAINED_GLASS, 0x999999);
-        put(Blocks.LIGHT_GRAY_STAINED_GLASS_PANE, 0x939393);
         put(Blocks.LIGHT_GRAY_TERRACOTTA, 0x876B62);
-        put(Blocks.LIGHT_GRAY_WALL_BANNER, 0x8F7748);
+        put(Blocks.LIGHT_GRAY_WALL_BANNER, 0x000000);
         put(Blocks.LIGHT_GRAY_WOOL, 0x8E8E87);
         put(Blocks.LIGHT_WEIGHTED_PRESSURE_PLATE, 0xF8D33E);
+        put(Blocks.LIGHT, 0x000000);
         put(Blocks.LIGHTNING_ROD, 0xC46F53);
         put(Blocks.LILAC, 0x977D8F);
         put(Blocks.LILY_OF_THE_VALLEY, 0x7DB061);
-        put(Blocks.LILY_PAD, 0x208030);
-        put(Blocks.LIME_BANNER, 0x8F7748);
+        put(Blocks.LILY_PAD, 0x888888);
```

Comment:
We need to color this ourselves. The texture is grey because Mojang uses 2 different green colors depending on where the lilypad is being drawn (inventory or world). I had picked the world color.